api_tutorials/bait_cnn/bait_performance_cnn.py

```diff
@@ -0,0 +1,528 @@
+    print('Loaded model weights from:', args.weights)
+
+    train_data = load_dna_and_bait_coverage(args)
+    train, valid, test = split_data(train_data)
+
+    title = weight_path_to_title(args.weights)
+
+    plot_scatter(model, [test.baitdata, test.gc], test.coverage, title)
+
+
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# ~~~~~~~ Models ~~~~~~~~~~~~~~~~~~
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+def build_small_sequential_bait_model(args):
+    model = Sequential()
+    model.add(Conv1D(input_dim=len(args.inputs),
+                     input_length=args.window_size,
+                     filters=40,
+                     filter_length=16,
+                     border_mode='valid',
+                     activation="relu",
+                     init='normal'))
+
+    model.add(MaxPooling1D(pool_length=3, stride=3))
+    model.add(Conv1D(nb_filter=64, filter_length=16, activation="relu", init='normal', border_mode='valid'))
+    model.add(Dropout(0.2))
+    model.add(MaxPooling1D(pool_length=3, stride=3))
+    model.add(Flatten())
+
+    model.add(Dense(output_dim=32, init='normal'))
+    model.add(Activation('relu'))
+
+    model.add(Dense(output_dim=1, init=RandomNormal(mean=1.0, stddev=0.5, seed=None)))
+    model.add(Activation(None))
+
+    sgd = SGD(lr=0.001, decay=1e-6, momentum=0.9, nesterov=True, clipnorm=0.5)
+    adamo = Adam(lr=0.001, beta_1=0.9, beta_2=0.999, epsilon=1e-08, clipnorm=1.)
+    my_metrics = [metrics.mean_squared_error, rmse_log]
+
+    model.compile(loss='mean_squared_error', optimizer=sgd, metrics=my_metrics)
+    print('model summary:\n', model.summary())
+
+    return model
+
+
+def build_small_functional_bait_model(args):
+    bait_shape = (args.window_size, len(args.inputs),)
+    annotation_shape = (len(args.annotations),)
+
+    print(bait_shape)
+    input_baits = Input(name="bait", shape=bait_shape)
+
+    x = Conv1D(activation='relu',
+               padding='valid',
+               filters=100,
+               kernel_size=8,
+               kernel_initializer='normal')(input_baits)
+
+    x = Dropout(0.2)(x)
+
+    x = MaxPooling1D(strides=3, pool_size=3)(x)
+
+    x = Conv1D(kernel_initializer="normal", activation="relu", padding="valid", filters=64, kernel_size=8)(x)
+    x = Dropout(0.2)(x)
+    x = MaxPooling1D(pool_size=3, strides=3)(x)
+
+    x = Conv1D(kernel_initializer="normal", activation="relu", padding="valid", filters=40, kernel_size=8)(x)
+    x = Dropout(0.2)(x)
+    x = MaxPooling1D(pool_size=3, strides=3)(x)
+
+    x = Flatten()(x)
+
+    x = Dense(units=32, kernel_initializer="normal", activation="relu")(x)
+
+    print(annotation_shape)
+
+    input_annotations = Input(name="annotation", shape=annotation_shape)
+
+    xy = Concatenate(axis=-1)([x, input_annotations])
+
+    xy = Dense(units=32, kernel_initializer="normal", activation="relu")(xy)
+
+    predictions = Dense(units=1, init=RandomNormal(mean=1.0, stddev=0.5, seed=None), activation=None)(xy)
```

Comment:
upon rereading I now understand. you're saying: "Yes, use a RELU in the last layer."